styleguide/source/assets/scss/00-base/_font-variables.scss

```diff
@@ -3,9 +3,9 @@ $myriad-pro: "myriad-pro", "Helvetica", "Arial", "Open Sans", sans-serif;
 $kepler-std: "kepler-std", Georgia, Times, "Times New Roman", serif;
 
 // Font weight variables
-$font-weight-regular: 100;
-$font-weight-semibold: 400;
-$font-weight-bold: 600;
+$font-weight-regular: 400;
+$font-weight-semibold: 600;
+$font-weight-bold: 900;
```

Comment:
It looks like this also updated the font weights on the article stubs in the column teaser and the four up teaser, were those also intended to be updated?